src/utils/web3.ts

```diff
@@ -19,6 +19,46 @@ export interface NetworkObject {
   urlList: string[]
 }
 
+const networks = [
```

Comment:
during refactor I tried to unify all network metadata getting so ideally we'll get rid of this variable definition, or fill it with data from somewhere else. 

As for the name itself, `getNetworkDisplayName()` is your friend, making sure we control network name from one central place across whole app.

Then I created `useNetworkMetadata()` which exposes metadata about all networks.

And finally I'm still trying to figure out how to get all `ConfigHelperConfig` out of ocean.js into either this network metadata hook, or as part of `useOcean`. Which we then can use to iterate over as those are the networks the switcher should provide. Ultimately we also have to cross-reference it with `appConfig.chainIdsSupported`. So we could already create some loop to iterate over `appConfig.chainIdsSupported` and construct the data with `getOceanConfig()`

 So this then together should have all the metadata we need without hardcoding them here again